src/misc/SvgPolyfill.ts

```diff
@@ -0,0 +1,11 @@
+import { InitializationEvents } from '../events/InitializationEvents';
+import { $$ } from '../utils/Dom';
+require('svg4everybody');
+
+declare function svg4everybody(): void;
+
+export function SvgPolyfill(root: HTMLElement) {
+  $$(root).on(InitializationEvents.afterComponentsInitialization, () => {
```

Comment:
It was needed because we need to wait for the components to initialize and therefore inline `use` tags.